src/Makefile.am

```diff
@@ -26,6 +26,9 @@ AM_CPPFLAGS += -I$(srcdir)/../apf
 ## Add GML path
 AM_CPPFLAGS += -I$(srcdir)/../gml/include
 
+## Add RapidJSON path
+AM_CPPFLAGS += -I$(srcdir)/../rapidjson/include
```

Comment:
We need to distinguish between system libs (if there are any) and the local ones.

Reply:
The reason why I didn't use the system lib is because Debian stable (https://packages.debian.org/stretch/rapidjson-dev) and Ubuntu xenial (which is the most modern Ubuntu available on Travis-CI, https://packages.ubuntu.com/xenial/rapidjson-dev) have a very old version of RapidJSON.

Do you have an idea how we could have both support for system libraries plus a git submodule as a fallback?

Or should we just require users to download RapidJSON by hand and provide the proper source directory?

The problem is that it is a header-only library, so it will not be installed in a default `include/` directory and I assume it isn't available for `pkg-config`.

Reply:
Oh, I see (man, those packages are old...).

I'm not quite sure why the Debian package for example doesn't provide pkg-config integration, but at least the headers are in the right place: https://packages.debian.org/stretch/all/rapidjson-dev/filelist

Usually, when installed to the system, rapidjson definitely provides a `pkg-config` integration via `/usr/lib/pkgconfig/RapidJSON.pc` (on Arch at least). Therefore we can use a check, such as `pkg-config --atleast-version 1.0.0 RapidJSON` and default back to the vendored rapidjson, in case the system can't match that requirement.
I assume we have to do some `IFDEF` magic for the includes to make this work, but it should not be super complicated.